src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -451,15 +451,16 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         FeedbackRubricQuestionDetails fqd = (FeedbackRubricQuestionDetails) question.getQuestionDetails();
         int[][] responseFrequency = calculateResponseFrequency(responses, fqd);
         float[][] rubricStats = calculateRubricStats(responses, question);
+        float[] average = calculateRubricAverage(responses, question);
```

Comment:
You can consider moving the average calculation to the same function `calculateRubricStats` For two reasons:
- Average _is_ part of the stats
- This nee function calls `calculateRubricStats`, i.e you're doing the same work twice in succession.